### PR TITLE
Changed Upload cancel dialog buttons value

### LINKED ISSUE #6588: Upload cancel dialog buttons incorrect
The buttons on the dialog that appears when you an tap an uploading media item in the editor should be YES / NO instead of OK / CANCEL.

uploading

### PR DESCRIPTION
**Pull request by Bayway**
Fixes #6588 

To test: the dialog appears when you tap an uploading media item in the editor.

### CODE REVIEW

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java [hunk 1]
```diff
@@ -1204,7 +1204,7 @@ private void onMediaTapped(@NotNull AztecAttributes attrs, int naturalWidth, int
                 // Display 'cancel upload' dialog
                 AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
                 builder.setTitle(getString(R.string.stop_upload_dialog_title));
-                builder.setPositiveButton(android.R.string.ok, new DialogInterface.OnClickListener() {
+                builder.setPositiveButton(getString(R.string.stop_upload_dialog_button_yes), new DialogInterface.OnClickListener() {
```
Comment (mzorz) [inline libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java]: can we use the `setPositiveButton(int,...`  version of the API? that is, ditch the `getString()` here and just pass the string resource id reference for `R.string. stop_upload_dialog_button_yes`.

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java [hunk 2]
```diff
@@ -1226,7 +1226,7 @@ public void onClick(DialogInterface dialog, int id) {
                     }
                 });
 
-                builder.setNegativeButton(getString(R.string.cancel), new DialogInterface.OnClickListener() {
+                builder.setNegativeButton(getString(R.string.stop_upload_dialog_button_no), new DialogInterface.OnClickListener() {
```
Comment (mzorz) [inline libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java]: same here

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java [hunk 3]
```diff
@@ -1339,7 +1339,7 @@ public void onMediaTapped(final String mediaId, final MediaType mediaType, final
                 // Display 'cancel upload' dialog
                 AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
                 builder.setTitle(getString(R.string.stop_upload_dialog_title));
-                builder.setPositiveButton(android.R.string.ok, new DialogInterface.OnClickListener() {
+                builder.setPositiveButton(getString(R.string.stop_upload_dialog_button_yes), new DialogInterface.OnClickListener() {
```
Comment (mzorz) [inline libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java]: here as well

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java [hunk 4]
```diff
@@ -1360,7 +1360,7 @@ public void run() {
                     }
                 });
 
-                builder.setNegativeButton(getString(R.string.cancel), new DialogInterface.OnClickListener() {
+                builder.setNegativeButton(getString(R.string.stop_upload_dialog_button_no), new DialogInterface.OnClickListener() {
```
Comment (mzorz) [inline libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java]: and here

**Comment by Bayway** [comment]
Sorry @mzorz but the italian string PR is a back commit, i had correct this and i push it for error. See here #6565

**Comment by mzorz** [comment]
> Sorry @mzorz but the italian string PR is a back commit, i had correct this and i push it for error. See here #6565

Ah I see! No problem, can you `git revert` that commit (a772549) and push again?

**Comment by mzorz** [comment]
hi @Bayway just a quick note to let you know it'd be good to go after these comments are addressed , i.e. https://github.com/wordpress-mobile/WordPress-Android/pull/6590#discussion_r135272383

**Comment by mzorz** [comment]
Awesome! Thank you @Bayway 🙇 
:shipit:

**Pull request merged**